README.md

```diff
@@ -116,12 +116,29 @@ The list of available actions can be seen via `medic-conf --help`.
 
 * upload of custom translations to the server
 
-## create-users **[ALPHA]**
+## create-users
 
-N.B. this feature is currently in development, and probably not ready for production yet.
 
 To create users on a remote server, use the `create-users` action.  The CSV file should be called `users.csv`, and an example is available [in the tests directory](test/data/create-users), for [an existing place](test/data/create-users/existing-place) and [a new place](test/data/create-users/new-place).
 
+### Linking users to contacts created from csv-to-docs
+
+To create user accounts for contacts that are created while running csv-to-docs action follow these steps.
+
+1. Create a users.csv file in the same folder as the rest of the csvs needed for `csv-to-docs` action.
```

Comment:
Should we just say "Create a `users.csv` file in your `json_docs` folder" since that folder is required by csv-to-docs?